qemu/tests/cfg/block_resize.cfg

```diff
@@ -55,6 +76,9 @@
     variants:
         - with_iozone:
             iozone_test = yes
+        - with_md5:
+            only default_partition
+            md5_test = yes
```

Comment:
For `with_gpt_ext4` , `with_msdos_xfs`, 'with_msdos_xfs' and `with_gpt_xfs`, don't do md5 testing?
CC @XueqiangWei @zhenyzha @peixiu @kimi1978wy

Reply:
There is iozone already exist and have better coverage than md5. We do not want to increase too many md5 cases.  It exist on  default_partition that is enough.